src/routes/Dashboard/SummaryCard/components/CounterSection.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from "prop-types";
 
 export default function CounterSection({value, label, addComma}) {
+    if(value===0 && label==="segundo") return "Iniciando..."
```

Comment:
bad formatting

Reply:
missing whitespaces

Reply:
Actually this works but is not a good idea.

The responsibility of this component is to render the counter, not to show a loading message.

The code looks like a monkey patch (hard-coded).

Do the fix in the place where this component is being called.